chef/roles/crowbar.rb

```diff
@@ -2,8 +2,7 @@
 name "crowbar"
 description "Crowbar role - Setups the rails app"
 run_list(
-         "recipe[utils]",
-         "recipe[crowbar]"
+         "recipe[crowbar::role_crowbar]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.